src/main/java/org/jenkinsci/plugins/durabletask/ProcessLiveness.java

```diff
@@ -96,12 +96,16 @@ private static boolean _isAlive(VirtualChannel channel, int pid, Launcher launch
             this.pid = pid;
         }
         @Override public Boolean call() throws RuntimeException {
-            // JNR-POSIX does not seem to work on FreeBSD at least, so using JNA instead.
```

Comment:
[diff for review](https://github.com/jenkinsci/durable-task-plugin/pull/47/files?w=1)